src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -115,11 +115,11 @@ public static function getStyleDefinition($name)
             self::$styles = self::initStyles();
         }
 
-        if (!self::$styles[$name]) {
-            throw new InvalidArgumentException(sprintf('Style "%s" is not defined.', $name));
+        if (isset(self::$styles[$name])) {
+            return self::$styles[$name];
```

Comment:
I'd keep the original order: throw, then return, which means changing this line to
`if (empty(self::$styles[$name])) {` (using `empty()` to keep throwing when defined but value is `== false`)